utils/protocol_registry.py

```
@@ -5,144 +5,152 @@
+
         # Save registry
-        with open(self.registry_file, 'w') as f:
+        with open(self.registry_file, "w") as f:
             json.dump(registry, f, indent=2)
-            
+
         log(f"Protocol {name} registered in category {category}")
         return True
-    
+
     def _calculate_checksum(self, protocol_name):
```

Ruff (0.11.9)

Missing return type annotation for private function `_calculate_checksum`

(ANN202)